client/my-sites/stats/feedback/index.tsx

```diff
@@ -47,6 +48,7 @@ function FeedbackContent( { clickHandler }: FeedbackProps ) {
 					{ secondaryButtonText }
 				</Button>
 			</div>
+			{ isOpen && <FeedbackModal siteId={ siteId } onClose={ () => setIsOpen( false ) } /> }
```

Comment:
Some code lost 🤔

Reply:
The `FeedbackModal` is part of the top-level controller component now. I'll have a look at fixing this bit.

Reply:
Should be fixed now.